### PR TITLE
Height-graph width reduced

### LINKED ISSUE #136: [BUG] HeightGraph Y-axis goes behind Directions Tab
### What is the problem you are facing
Users when trying to find Directions using the Directions tab and at the same time want to look at the height graph, then some part of the graph is not visible. Also when the user closes that tab, the graph tab also closes automatically. 

### What is your suggested solution
We can improve user experience by solving this problem. We can either reduce the size of the graph permanently or we can reduce its width only for the time when the tab is open. This thing can be achieved by managing states. I think the latter idea would be better, though managing states is a bit difficult for me here. If anyone has already managed states in it please tell me. 

### Screenshots

image8

### PR DESCRIPTION
**Pull request by MihirKp25**
## 🛠️ Fixes Issue
Closes #136 

## 👨‍💻 Changes proposed

I have reduced the width of height graph from 0.9 to 0.75 in Map.jsx file. I tried to change this width dynamically but it didn't end well. But still it looks pretty nice to me and now we can easily see y axis of height graph.

## 📄 Note to reviewers

If you find a way of reducing the width dynamically whenever the direction tab closes please tell me :)

## 📷 Screenshots
image9

### CODE REVIEW

**Comment by chrstnbwnkl** [comment]
@MihirKp25 could you please resolve existing conflicts so we can have a look at your PR?

**Comment by MihirKp25** [comment]
I have resolved conflicts. Sorry for inconvenience.

**Review by nilsnolde — CHANGES_REQUESTED** [review]
The whitespace-only changes seem irrelevant, I’d actually expect prettier to complain.. Can you revert those?

File: src/Controls/Directions/Waypoints/Waypoint.jsx [hunk 1]
```diff
@@ -139,7 +139,6 @@ class Waypoint extends React.Component {
               </Label>
             }
           />
-
```
Comment (chrstnbwnkl) [inline src/Controls/Directions/Waypoints/Waypoint.jsx:142]: please check your PR again for unnecessary, whitespace-only changes.

**Review by chrstnbwnkl — CHANGES_REQUESTED** [review]

File: src/Controls/Directions/Waypoints/Waypoint.jsx (hunk 1 above)
Comment (MihirKp25) [reply src/Controls/Directions/Waypoints/Waypoint.jsx]: Sorry I misinterpreted the last change request and added spaces in this file. I removed those changes. Please let me know if there are any other unnecessary changes I have made. And sorry for the inconvenience.

**Review by nilsnolde — APPROVED** [review]
Fine for me, thanks. I'll let @chrstnbwnkl merge if he's ok with this.

**Comment by MihirKp25** [comment]
I will try to make your suggested changes asap.

**Comment by chrstnbwnkl** [comment]
@MihirKp25 can you push your latest changes? The functionality in the video looks pretty close to what we want to achieve, right?

**Comment by MihirKp25** [comment]
Changes are pushed @chrstnbwnkl

**Comment by nilsnolde** [comment]
This looks pretty good, thanks @MihirKp25 ! 

You can find more info on the proposal here: https://wiki.openstreetmap.org/wiki/Google_Summer_of_Code/2023 and here: https://github.com/gis-ops/valhalla-app/issues/67. If you follow those guidelines you should be good. We have (un)fortunately too many applicants to do extensive proposal review. I suggest you upload it to GSoC, we'll surely read and consider it.

**Review by nilsnolde — APPROVED** [review]

**Pull request merged**